examples/demo/gloo/quiver.py

```diff
@@ -264,6 +264,13 @@
 
 
 @canvas.connect
+def on_initialize(event):
+    gl.glEnable(gl.GL_BLEND)
+    gl.glBlendFunc(gl.GL_SRC_ALPHA, gl.GL_ONE_MINUS_SRC_ALPHA)
+    gl.glBlendEquation(gl.GL_FUNC_ADD)
```

Comment:
better to use `set_state` here instead. So it would be:
```
canvas.context.set_state(blend=True, blend_func=('src_alpha', 'one_minus_src_alpha'), blend_equation='func_add'))
```